Tests/GitHubReleases.tests.ps1

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.Synopsis
+   Tests for GitHubLabels.ps1 module
```

Comment:
```suggestion
   Tests for GitHubReleases.ps1 module
```

Reply:
Should I hit **Resolve conversation** on all of these?